app/com/linkedin/drelephant/util/Utils.java

```diff
@@ -234,6 +237,79 @@ public static String truncateField(String field, int limit, String appId) {
   }
 
   /**
+   * Convert a millisecond duration to a string format
+   *
+   * @param millis A duration to convert to a string form
+   * @return A string of the form "X:Y:Z Hours".
+   */
+  public static String getDurationBreakdown(long millis) {
+
+    long hours = TimeUnit.MILLISECONDS.toHours(millis);
+    millis -= TimeUnit.HOURS.toMillis(hours);
+    long minutes = TimeUnit.MILLISECONDS.toMinutes(millis);
+    millis -= TimeUnit.MINUTES.toMillis(minutes);
+    long seconds = TimeUnit.MILLISECONDS.toSeconds(millis);
+
+    StringBuilder sb = new StringBuilder(64);
+
+    sb.append(hours);
+    sb.append(":");
+
+    sb.append(minutes);
+    sb.append(":");
+
+    sb.append(seconds);
+    sb.append(" Hours");
+
+    return (sb.toString());
+  }
+
+  /**
+   * Convert a value in MBSeconds to GBHours
+   * @param MBSeconds The value to convert
+   * @return A string of form a.xyz GB Hours
+   */
+  public static String getDurationInGBHours(long MBSeconds) {
+
+    if (MBSeconds == 0) {
+      return "0 GB Hours";
+    }
+    double GBseconds = (double) MBSeconds / (double) Statistics.KB;
+    double GBHours = GBseconds / Statistics.HOUR;
+
+    if ((long) (GBHours * 1000) == 0) {
+      return "0 GB Hours";
+    }
+
+    DecimalFormat df = new DecimalFormat("#.000");
+    String GBHoursString = df.format(GBHours);
+    GBHoursString = GBHoursString + " GB Hours";
+
+    if (GBHoursString.startsWith(".")) {
```

Comment:
I think if we set the DecimalFormat as 0.000, then we don't need to append 0? right?